daemon/daemon_unix.go

```diff
@@ -155,7 +160,7 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 	if hostConfig.Memory == 0 && hostConfig.MemorySwap > 0 {
 		return warnings, fmt.Errorf("You should always set the Memory limit when using Memoryswap limit, see usage.")
 	}
-	if hostConfig.MemorySwappiness != nil && !daemon.SystemConfig().MemorySwappiness {
+	if hostConfig.MemorySwappiness != nil && !sysInfo.MemorySwappiness {
```

Comment:
@hqhq now I get
```
./bundles/1.8.0-dev/binary/docker run -d debian:jessie true
WARNING: Your kernel does not support memory swappiness capabilities, memory swappiness discarded.
e8076e48660a6f7bded2ab67fb59a11c1878af79d78dff636957898e2803971e
```
but I suppose this is the correct behavior because Swappiness default value is -1